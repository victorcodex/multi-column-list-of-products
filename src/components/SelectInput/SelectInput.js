import styled from "styled-components";

const SelectInput = styled.select`
    position: relative;
    height: 4vh;
    width: 100%;
    border-radius: 1vh;
    border: 1px solid #000000;
    color: #3a2b2b;
    background: #f3f3f3;
    padding: 0;
    text-align: left;
    cursor: pointer;
    outline:0;
`;

const SelectInputLabel = styled.div`
    width: 25%
`;

export {
    SelectInput,
    SelectInputLabel
}