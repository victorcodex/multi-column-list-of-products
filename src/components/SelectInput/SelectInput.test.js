import React from 'react';
import { render } from '@testing-library/react';
import { SelectInput } from './SelectInput';

describe('SelectInput component', () => {
  it('should render select element', () => {
    const { asFragment } = render(<SelectInput />);
    expect(asFragment()).toMatchSnapshot();
  });
});