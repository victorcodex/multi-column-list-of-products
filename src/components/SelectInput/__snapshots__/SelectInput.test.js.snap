// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SelectInput component should render select element 1`] = `
<DocumentFragment>
  <select
    class="sc-bdnxRM koaRiL"
  />
</DocumentFragment>
`;
